packages/medusa/src/services/cache.ts

```diff
@@ -0,0 +1,67 @@
+import { Redis } from "ioredis"
```

Comment:
todo(non-blocking): Can I get you to create a chore issue on replacing the cache implementation in the tax provider to ensure we don't end up with duplicated code?

Reply:
Additionally, I think we should tackle that same issue asap to not let it be forgotten.

Reply:
I think that it should be replaced everywhere in the core in fact. 
The purpose of that service is to have a first step of removing the notion of redis in the service directly and be localised in one place for later extraction. At the moment, the tax provider is the only one that does no reference redis directly unlike the other places where the cache is used.
What do you think of a linear issue in two steps:
1. replace the cache usage to use the cache service
2. extract it to a module